features/org.wso2.carbon.identity.oauth.server.feature/resources/oauth-scope-bindings.xml

```diff
@@ -886,4 +886,54 @@
         <Permission>/permission/admin/manage/identity/cors/origins/view</Permission>
     </Permissions>
 </Scope>
+    <Scope name="internal_scope_mgt_view" displayName="View Scope">
+        <Permissions>
+            <Permission>/permission/admin/manage/identity/scopemgt/view</Permission>
+        </Permissions>
+    </Scope>
+    <Scope name="internal_scope_mgt_create" displayName="Create Scope">
+        <Permissions>
+            <Permission>/permission/admin/manage/identity/scopemgt/create</Permission>
+        </Permissions>
+    </Scope>
+    <Scope name="internal_scope_mgt_update" displayName="Update Scope">
+        <Permissions>
+            <Permission>/permission/admin/manage/identity/scopemgt/update</Permission>
+        </Permissions>
+    </Scope>
+    <Scope name="internal_scope_mgt_delete" displayName="Delete Scope">
+        <Permissions>
+            <Permission>/permission/admin/manage/identity/scopemgt/delete</Permission>
+        </Permissions>
+    </Scope>
+    <Scope name="internal_governance_view" displayName="View Governance Connector">
+        <Permissions>
+            <Permission>/permission/admin/manage/identity/governance/view</Permission>
+        </Permissions>
+    </Scope>
+    <Scope name="internal_governance_update" displayName="Update Governance Connector">
+        <Permissions>
+            <Permission>/permission/admin/manage/identity/governance/update</Permission>
+        </Permissions>
+    </Scope>
+    <Scope name="internal_group_mgt_create" displayName="Create Groups">
+        <Permissions>
+            <Permission>/permission/admin/manage/identity/groupmgt/create</Permission>
+        </Permissions>
+    </Scope>
+    <Scope name="internal_group_mgt_delete" displayName="Delete Groups">
+        <Permissions>
+            <Permission>/permission/admin/manage/identity/groupmgt/delete</Permission>
+        </Permissions>
+    </Scope>
+    <Scope name="internal_group_mgt_view" displayName="View Groups">
+        <Permissions>
+            <Permission>/permission/admin/manage/identity/groupmgt/view</Permission>
+        </Permissions>
+    </Scope>
+    <Scope name="internal_group_mgt_update" displayName="Update Groups">
+        <Permissions>
+            <Permission>/permission/admin/manage/identity/groupmgt/update</Permission>
+        </Permissions>
+    </Scope>
```

Comment:
As a part of the restructuring effort for 5.12.0 on-prem docs, we have documented scopes corresponding to permissions required to invoke REST API calls at https://github.com/wso2/docs-is/blob/new_restructure/en/docs/references/scopes-corresponding-to-rest-api-permissions.md
Therefore let's create a doc issue to track this addition.